community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -316,9 +316,11 @@ private static String defaultPageCacheMemory()
                 long physicalMemory = (long) getTotalPhysicalMemorySize.invoke( os );
                 if ( 0 < physicalMemory && physicalMemory < Long.MAX_VALUE && maxHeapMemory < physicalMemory )
                 {
-                    long heuristic = (long) ((physicalMemory - maxHeapMemory) * 0.75);
-                    long min = 32 * 1024 * 1024; // We'd like at least 32 MiBs.
-                    long max = 1024 * 1024 * 1024 * 1024L; // Don't heuristically take more than 1 TiB.
+                    double ratioOfFreeMem = 0.50;
+                    long heuristic = (long) ((physicalMemory - maxHeapMemory) * ratioOfFreeMem);
+                    long MiB = 1024 * 1024;
```

Comment:
do not really see a problem with a language expressiveness
 Anyway, do we need to convert from one unit to another? if no, then would think that enum is a bit overkill